.github/workflows/workflow.yml

```diff
@@ -0,0 +1,50 @@
+name: AEA framework sanity checks and tests
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+
+jobs:
+  run:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@master
+    - uses: actions/setup-python@master
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        pip install pipenv
+        pip install tox
+    - name: Security Check - Main
+      run: tox -e bandit-main
+    - name: Security Check - Tests
+      run: tox -e bandit-tests
+    - name: Code style check
+      run: |
+        tox -e black-check
+        tox -e flake8
+    - name: Static type check
+      run: tox -e mypy
+    - name: Unit tests and coverage
+      run: |
+        tox -e py${{ matrix.python-version }} -- --no-integration-tests
```

Comment:
@marcofavorito neither I nor @cyenyxe did know why the current Jenkinsfile has the `--ci` flag. Do you?